p2p/src/net/default_backend/transport/impls/channel.rs

```diff
@@ -74,28 +74,30 @@ static CONNECTIONS: Lazy<Mutex<BTreeMap<Address, UnboundedSender<IncomingConnect
 
 static NEXT_ADDRESS: AtomicU32 = AtomicU32::new(1);
 
-// Creating new transport is like attaching new "host" to the network.
-// New unique address is registered for the new "host".
-// Unlike TCP only one active bind is allowed at any moment to keep things simple
-// (so there is only one port per host).
+/// Creating new transport is like attaching new "host" to the network.
+/// New unique address is registered for the new "host".
+/// Unlike TCP only one active bind is allowed at any moment to keep things simple
+/// (so there is only one port per host).
+///
+/// This transport should only be used in tests.
 #[derive(Debug)]
-pub struct MockChannelTransport {
+pub struct TestChannelTransport {
     local_address: Address,
 }
 
-impl MockChannelTransport {
+impl TestChannelTransport {
     pub fn new() -> Self {
         let local_address = NEXT_ADDRESS.fetch_add(1, Ordering::Relaxed);
-        MockChannelTransport { local_address }
+        TestChannelTransport { local_address }
     }
 }
 
 #[async_trait]
-impl TransportSocket for MockChannelTransport {
+impl TransportSocket for TestChannelTransport {
```

Comment:
I would call this MpscChannelTransport? Feel free to suggest a better name. The fact that it's for testing doesn't need to be in the name.